distribution/changelog.txt

```diff
@@ -1,5 +1,6 @@
-0.0.8 (in development)
+﻿0.0.8 (in development)
```

Comment:
The UTF-8 byte order mark, try using Notepad++ or a similar text editor to remove the BOM.